boards/arm/nucleo_l152re/pinmux.c

```diff
@@ -24,6 +24,9 @@ static const struct pin_config pinconf[] = {
 #if DT_HAS_NODE_STATUS_OKAY(DT_NODELABEL(adc1))
 	{STM32_PIN_PA0, STM32L1X_PINMUX_FUNC_PA0_ADC1_IN0},
 #endif
+#if DT_HAS_NODE_STATUS_OKAY(DT_NODELABEL(dac1))
```

Comment:
Needs to be updated to `#if DT_NODE_HAS_STATUS(DT_NODELABEL(dac1), okay) && CONFIG_DAC`

Reply:
Done, thanks!